package.json

```diff
@@ -75,7 +75,7 @@
     "@atlaskit/pagination": "^8.0.5",
     "@types/micromatch": "^4.0.1",
     "atlassian-jwt": "^1.0.1",
-    "axios": "^0.19.2",
+    "axios": "^0.21.1",
```

Comment:
Just for my information - is there any major changes between those 2 minor versions?

Reply:
It's all fixes, integration tests worked fine and they use all the axios functionality that Landkid uses.

https://github.com/axios/axios/releases